File: woodroid/src/main/java/me/gilo/woodroid/utils/Converter.java
package me.gilo.woodroid.utils;

import java.text.SimpleDateFormat;
import java.util.Date;

public class Converter {

    public static String getDateString(Date date){
        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX");
        return  sdf.format(date);
    }
}

File: woodroid/src/main/java/me/gilo/woodroid/utils/Converter.kt
package me.gilo.woodroid.utils

import java.text.SimpleDateFormat
import java.util.Date
import java.util.Locale

object Converter {

    fun getDateString(date: Date): String {
        val sdf = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX", Locale.getDefault())
        return sdf.format(date)
    }
}
